manifests/cf-manifest/deployments/aws/060-aws-broker-stub.yml

```diff
@@ -0,0 +1,18 @@
+resource_pools:
+  - name: rds_broker_z1
+    network: cf1
+    stemcell: (( grab meta.stemcell ))
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.medium
+      ephemeral_disk:
+        size: 10240
+        type: gp2
+      availability_zone: (( grab meta.zones.z1 ))
+      iam_instance_profile: cf-cloudcontroller
```

Comment:
Does this need an instance profile at all then? I think it would be better to not set one than to set this one, which would be a little misleading when it comes to the next story.